tests/inspect/inspect.go

```diff
@@ -194,6 +196,42 @@ func main() {
 		fmt.Printf("Memory Limit: %s\n", string(limit))
 	}
 
+	if globalFlags.PrintCPUQuota {
+		cpuCgroupPath, err := getOwnCgroupPath("cpu")
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Error getting own cpu cgroup path: %v\n", err)
+			os.Exit(1)
+		}
+		// we use /proc/1/root to escape the chroot we're in and read our
+		// cpu quota
+		periodPath := filepath.Join("/proc/1/root/sys/fs/cgroup/cpu", cpuCgroupPath, "cpu.cfs_period_us")
```

Comment:
On Debian-Sid, I have `cpu.cfs_quota_us` but not `cpu.cfs_period_us`. rkt does not check for `cpu.cfs_period_us`:

```
        cgroupControllerRWFiles = map[string][]string{
                "memory": []string{"memory.limit_in_bytes"},
                "cpu":    []string{"cpu.cfs_quota_us"},
        }
```

I wonder if you should consider the default value: `cpu.cfs_period_us=100ms`. But let's find out why Debian does not have this file...

```
root@alban:/sys/fs/cgroup/cpu# ls -l cpu.*
-rw-r--r-- 1 root root 0 Jun  4 09:58 cpu.shares
```